app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+###
+#  User class for authenticating via WebAuth
+###
+class User
+  attr_accessor :webauth
+
+  def initialize(webauth)
+    @webauth = webauth
+  end
+
+  def webauth_user?
+    @webauth.present?
```

Comment:
@jgreben Just so I'm clear, you're saying that changing line 12 in this file from `@webauth.present?` to `webauth.present?` doesn't work?